config.yaml

```diff
@@ -17,21 +27,16 @@ common:
    --plugin-option=tmpfs:keep_mounted=True
    --plugin-option=tmpfs:max_fs_size=32g
    --plugin-option=tmpfs:required_ram_mb=39800"
- commit_updates: True
- push_updates: True
-build_packages:
- packages:
- packages_repos_target_path: "/var/lib/host-os/repositories"
- keep_build_dir: False
- result_dir:  "./result"
- force_rebuild: False
 build_release_notes:
  repo_url: "https://github.com/open-power-host-os/open-power-host-os.github.io.git"
  repo_branch: "master"
+ push_repo_url: "git@github.com:open-power-host-os/open-power-host-os.github.io.git"
```

Comment:
Do not set this default value. We shouldn't push directly to the main repository (it may happen by accident) and any other user will not have permission to do so. We should make it explicit that this parameter is required and needs to be set up properly.